internal/services/instance_types_service.go

```diff
@@ -16,7 +16,12 @@ import (
 
 func ListInstanceTypes(w http.ResponseWriter, r *http.Request) {
 	sourceId := chi.URLParam(r, "ID")
-	ec2Client := ec2.NewEC2Client(r.Context())
+
+	region := r.URL.Query().Get("region")
+	if region == "" {
+		renderError(w, r, payloads.NewNotFoundError(r.Context(), ec2.RegionNotFoundErr))
```

Comment:
For easier transition, we could allow for empty region for the transition period (until the UI part would be merged).
But given we do not have UI pipeline yet, there is no need, just thinking out loud how to do this in the future ^_^
No action necessary here, I just needed to say this xD